wallet-new/src/Cardano/Wallet/Kernel/Accounts.hs

```diff
@@ -71,14 +67,14 @@ instance Exception CreateAccountError
 --
 
 data DeleteAccountError =
-      DeleteAccountUnknownHdRoot HdRootId
-      -- ^ When trying to delete the 'Account', the parent 'HdRoot' was not
-      -- there.
+      DeleteAccountError UnknownHdAccount
```

Comment:
We reviewed this with @edsko , we think this is not there yet: what we should be doing perhaps is to return the unaltered Hd-specific error upstream and then re-wrap this error in a `V1` coating in the WalletLayer specific error. this way we make the distinction clearer.